src/main/java/it/cnr/isti/hpc/wikipedia/article/Link.java

```diff
@@ -100,7 +132,7 @@ public int hashCode() {
 		final int prime = 31;
 		int result = 1;
 		result = prime * result
-				+ ((description == null) ? 0 : description.hashCode());
+				+ ((anchor == null) ? 0 : anchor.hashCode());
```

Comment:
object == null is fine, but I'm not sure about the hashcode function.. did you write it or did you autogenerate?